VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -219,10 +219,11 @@ private void bindViewHolderToResponseRow(DiscussionResponseViewHolder holder, fi
             @Override
             public void onClick(View v) {
                 if (comment.getChildren().isEmpty()) {
-                    router.showCourseDiscussionAddComment(context, comment);
+                    if (!discussionThread.isClosed())
+                        router.showCourseDiscussionAddComment(context, comment);
```

Comment:
>Also, should the add comment button even be visible if commenting is closed?

Yes, here's how it looks:
![closed](https://cloud.githubusercontent.com/assets/1710804/10310835/666c0658-6c5d-11e5-9e48-811ad9e4a636.jpg)